wire-ios-transport/Source/Public/BackendInfo.swift

```diff
@@ -29,7 +29,10 @@ public enum BackendInfo {
 
     }
 
-    public static var storage = UserDefaults.standard
+    private static let productionStorage = UserDefaults.standard
```

Comment:
Is `.standard` really the "production" value?
In `LaunchSequenceOperation.swift` I see the line
```
BackendInfo.storage = .applicationGroup
```

What about not setting any default value at all?
```suggestion
    private static let productionStorage: UserDefaults!
```

Reply:
Interesting. It looks like that is the only place it is being overridden and it is being done by the `AutomationHelper` which I assume is for UI tests. I'll take a close look.

Reply:
I tried using `UserDefaults!` but this causes crashes in tests as we must still be implicitly depending on `BackendInfo` in many places. Therefore, I took a different approach where we return `mockStorage` if it is set, otherwise we return `storage`.